app/test/datastore_test.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import boto3
+from moto import mock_dynamodb2
+import sys
+sys.path.append("../lambda/")
+from datastore import DataStore
+
+class TestDynamo(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    @mock_dynamodb2
+    def test_recoverBsaleAssociation(self):
```

Comment:
Is there a specific reason to this name?

Also, lets use 'test' as the prefix for both function name and file name.
The file name could be test_datastore.py

All your tests on datastore would require you to create a dynamodb table so you could might as well create that separately as a part of *setUp* steps and delete the tables as a part of *tearDown* steps. More info on these available [here](https://docs.python.org/3/library/unittest.html#module-unittest)

The tests should ideally correspond to the functions defined in Datastore class.
Hence the functions should be test_save, test_list_items etc.

Reply:
Got it!